src/main/cli/cli.c

```diff
@@ -4876,21 +4878,24 @@ typedef struct dmaoptEntry_s {
 // DEFW : Wider stride case; array of structs.
 
 #define DEFS(device, peripheral, pgn, type, member) \
-    { device, peripheral, pgn, 0,               offsetof(type, member), 0 }
+    { device, peripheral, pgn, 0,               offsetof(type, member), 0, 0 }
```

Comment:
perhaps `...NO_MASK` so that readers don't have to look up the what the second `, 0` means?

Reply:
Defined and used `MASK_IGNORED` to emphasize the field is  not used, and used the same symbol when checking the presenceMask.